inductiva/tasks/task.py

```diff
@@ -152,25 +157,91 @@ def get_output(self, output_dir=None):
         Returns:
             The output of the task.
         """
-
-        # Blocking call for the task to terminate
         self.wait()
-        _, output_dir = api.download_output(self._api, self.id, output_dir)
-        output_dir = files.resolve_path(output_dir)
+        output_dir = self.download_outputs(output_dir)
 
         if self._output_class:
             return self._output_class(output_dir)
 
         return output_dir
 
-    def download_output(self, output_dir=None) -> pathlib.Path:
-        """Download the output of the task.
+    def get_outputs_info(self):
+        """Get information of the output files of the task.
 
         Returns:
-            The path to the downloaded output directory.
+            An instance of the OutputContents class, which can be used to
+            access info about the output files, such as the size of archive,
+            number of files, and information about each file (name, size,
+            compressed size). It can also be used to print that information
+            in a formatted way.
```

Comment:
What do you mean by " It can also be used to print that information in a formatted way."? Isn't it always printed in a formatted way?